src/asn1/asn1-cerstream.cc

```diff
@@ -154,51 +166,61 @@ Deserializer& Deserializer::operator>>(const Token& tok) {
       seq_consumed.pop();
       if (len >= 0) {
         while (len > cons_len) {
-          if (cer_decode_token(data, &endpos, nullptr, nullptr) == kUnknown) throw deserialization_error();
+          if (cer_decode_token(data, &endpos, nullptr, nullptr) == kUnknown) {
+            throw deserialization_error();
+          }
           data = data.substr(endpos);
           cons_len += endpos;
         }
-        if (cons_len != len) throw deserialization_error();
+        if (cons_len != len) {
+          throw deserialization_error();
+        }
       } else {
         int nestedness = 0;
         int32_t int_param = 0;
 
         for (;;) {
-          if (data.empty())  // end of data before end of sequence
+          if (data.empty()) {  // end of data before end of sequence
             throw deserialization_error();
+          }
           uint8_t ret = cer_decode_token(data, &endpos, &int_param, nullptr);
           data = data.substr(endpos);
           cons_len += endpos;
 
           if (ret == kUnknown) {
             throw deserialization_error();
-          } else if (ret == kAsn1Sequence) {  // TODO: explicit tags (constructed tokens in general)
+          }
+          if (ret == kAsn1Sequence) {  // TODO(lbonn): explicit tags (constructed tokens in general)
             // indefininte length, expect an end of sequence marker
-            if (int_param == -1) ++nestedness;
+            if (int_param == -1) {
+              ++nestedness;
+            }
           } else if (ret == kAsn1EndSequence) {
-            if (nestedness == 0)
+            if (nestedness == 0) {
               break;
-            else
-              --nestedness;
+            }
+            { --nestedness; }
```

Comment:
Why is this in brackets?

Reply:
because i"f else" should be always with brackets. Without breckets it is easy to write an incorrect logic

Reply:
So it should look like this?

```
if (nestedness == 0) {
  break;
}
{ --nestedness; }
```

That just seems odd to me. It won't hurt anything, but I don't get it. If there was an `else` it would make sense.

Reply:
(For the record, I'm 100% in favor of adding the brackets for all ifs and elses, but this case is a little weirder to me.)